edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.Test/edgedeployment/EdgeDeploymentOperatorTest.cs

```diff
@@ -216,8 +215,7 @@ public async void StatusIsFailedWithSpecialMessageWhenHttpExceptionIsThrown()
                 client,
                 controller);
 
-            HttpOperationException ex = await Assert.ThrowsAsync<HttpOperationException>(() => edgeOperator.EdgeDeploymentOnEventHandlerAsync(WatchEventType.Added, edgeDefinition));
-            Assert.Equal(controllerException, ex);
+            await edgeOperator.EdgeDeploymentOnEventHandlerAsync(WatchEventType.Added, edgeDefinition);
             Assert.True(reportedStatus.HasValue);
             Assert.Equal(expectedStatus, reportedStatus.OrDefault());
```

Comment:
Given there is failure deploymentstatus, can we run the deploymentstatuscommand and verify? Same for the other one.

Reply:
That's more of an integration test, but I can add one.

Reply:
It's just we are eating exception. Or we can change the test case title.